astropy/units/quantity_helper/function_helpers.py

```diff
@@ -598,10 +597,10 @@ def histogram(a, bins=10, range=None, normed=None, weights=None,
     if not isinstance(bins, str):
         bins = _check_bins(bins, a.unit)
 
-    if density or normed:
+    if density:
         unit = (unit or 1) / a.unit
 
-    return ((a.value, bins, range, normed, weights, density), {},
+    return ((a.value, bins, range, None, weights, density), {},
```

Comment:
To future-proof this for the eventual removal of `normed`, let's do `((a.value, bins, range), {'weights': weights, 'density': density}, ...` (here and below)

Reply:
Ahhh, so that's how it hooks up to Numpy API. Gotcha. Done!